shell/platform/android/platform_view_android.cc

```diff
@@ -126,34 +127,28 @@ static std::unique_ptr<AndroidSurface> InitializePlatformSurface() {
 }
 
 PlatformViewAndroid::PlatformViewAndroid()
-    : PlatformView(std::make_unique<GPURasterizer>(nullptr)),
-      android_surface_(InitializePlatformSurface()) {}
+    : PlatformView(std::make_unique<NullRasterizer>()),
+      android_surface_(nullptr) {}
```

Comment:
Would it be possible to continue creating the AndroidSurface at PlatformView construction time?

The AndroidSurface's lifetime would remain tied to that of the PlatformView, but the AndroidSurface would gain and lose the onscreen context as the EGL surface is created and destroyed.

Reply:
Done